tasks/main.yml

```diff
@@ -34,8 +34,14 @@
     group: "{{ az_devops_agent_user }}"
     creates: "{{ az_devops_agent_folder }}/config.sh"
 
+- name: Install Dependencies via shipped script
+  command: ./bin/installdependencies.sh
+  become: true
+  args:
+    chdir: "{{ az_devops_agent_folder }}"
+
 - name: Register agent service status
-  shell: ./svc.sh status | grep -q 'not installed'
+  shell: if [ ! -f "/etc/systemd/system/$(systemd-escape vsts.agent.akelius.{{ az_devops_agent_pool_name }}.{{ azure_virtual_machine_name }}.dservice)" ]; then true; else false; fi
```

Comment:
`vsts.agent.akelius` seems like you forgot to remove your account name. Pleace replace with provided variable `az_devops_accountname`.
Same for `azure_virtual_machine_name`.  You should be using `az_devops_agent_name` here, especially since the host isn't constrainted to be hosted in Azure ;)

Reply:
Also maybe look into multilining that shell script for readability. 
See https://stackoverflow.com/questions/40230184/how-to-do-multiline-shell-script-in-ansible

Reply:
corrected